query/sources/impl/src/test/groovy/org/codice/ddf/admin/sources/csw/discover/DiscoverCswSourceSpec.groovy

```diff
@@ -0,0 +1,271 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ **/
+package org.codice.ddf.admin.sources.csw.discover
+
+import org.codice.ddf.admin.api.fields.FunctionField
+import org.codice.ddf.admin.common.fields.common.HostField
+import org.codice.ddf.admin.common.report.message.DefaultMessages
+import org.codice.ddf.admin.sources.csw.CswSourceUtils
+import org.codice.ddf.admin.sources.fields.CswProfile
+import org.codice.ddf.admin.sources.fields.type.CswSourceConfigurationField
+import org.codice.ddf.admin.sources.test.SourceCommonsSpec
+import spock.lang.Shared
+
+class DiscoverCswSourceSpec extends SourceCommonsSpec {
+
+    @Shared
+            ddfCswResponse = this.getClass().getClassLoader().getResource('responses/csw/DDFCswGetCapabilities.xml').text
+
+    @Shared
+            specCswResponse = this.getClass().getClassLoader().getResource('responses/csw/specCswGetCapabilities.xml').text
+
+    @Shared
+            gmdCswResponse = this.getClass().getClassLoader().getResource('responses/csw/gmdCswGetCapabilities.xml').text
+
+    @Shared
+            unrecognizedCswResponse = this.getClass().getClassLoader().getResource('responses/csw/unrecognizedCswSchema.xml').text
+
+    @Shared
+            noOutputSchemaCswResponse = this.getClass().getClassLoader().getResource('responses/csw/noOutputSchemaResponse.xml').text
+
+    @Shared
+            badResponseBody = this.getClass().getClassLoader().getResource('responses/badResponse.xml').text
+
+    static TEST_CSW_URL = 'https://localhost:8993/services/csw'
+
+    static NO_FILTER = 'NO_FILTER'
+
+    DiscoverCswSource discoverCsw
```

Comment:
❗️ Please move this field down so it's not in the middle of your statics. You might also want to move all the static fields up above the shared fields.